### ISSUE TITLE
Microsoft Teams AzureBot is unresponsive

### ISSUE THREAD

**Issue by mikejonestechno** [Microsoft/AzureBot#135]
Just started using Microsoft Teams this week. I opened Chat, 'Discovered Bots' and added the AzureBot which now shows up in the Chat panel.

I can enter messages to AzureBot but there is no response - or error messages. I cannot find any way to confifgure or try re-installing the bot. Is AzureBot a hosted service or do I have to clone and host it myself?

![azurebot unresponsive](https://cloud.githubusercontent.com/assets/16675734/24078184/e1822c6a-0cba-11e7-8cbc-b63b7621ba34.PNG)

**Comment by nzthiago** [comment]
The Azure Bot is a sample that we host and maintain it on a best effort basis. You might have caught it right as we were fixing this issue with updating its Cognitive Services key: https://github.com/Microsoft/AzureBot/issues/133
Try again and let us know. It seems to be working for me in Microsoft Teams and other channels now.
![image](https://cloud.githubusercontent.com/assets/1563288/24225005/7575fe1c-0f1b-11e7-8cd3-ac4ec1eaba9c.png)

**Comment by mikejonestechno** [comment]
Still unresponsive, doesn't display any options or responses. I can't see any way to configure or debug the bot. 

Any suggestions to try?

![azurebotunresponsive](https://cloud.githubusercontent.com/assets/16675734/24226704/0f0d64c0-0fbc-11e7-92c7-99fd9ba7755f.PNG)

**Comment by nzthiago** [comment]
Hmm, that is quite strange indeed. If you open this URL does it take you to the same AzureBot instance?
https://teams.microsoft.com/l/chat/0/0?users=28:9a38affe-4fa0-493c-b802-432a83d8842f

Also, if you send "/deleteprofile" does it still not say anything? It should reply with this:

![image](https://cloud.githubusercontent.com/assets/1563288/24253071/861663b6-0f9c-11e7-9651-4a3c870bb56f.png)

**Comment by mikejonestechno** [comment]
That link takes me to the same AzureBot chat instance in Microsoft Teams app. I've also clicked the link and opened in Chrome browser - took me to the same chat instance with the history of unresponsive messages.

I tried posting a login message from the app on my home PC in case there was some corporate network connectivity issue with office laptop (at 8.29pm local time) but still no response in the conversation.

Is there any way to remove the bot or debug connectivity? I would have expected some response if the bot was offline or unavailable?

Thanks for your help @nzthiago 

![azurebot-unresponsive](https://cloud.githubusercontent.com/assets/16675734/24279243/be3b7352-109c-11e7-9d57-9a2c581c8301.PNG)
